BubblesKotlin/app/src/main/java/com/example/android/bubbles/data/NotificationHelper.kt

```diff
@@ -105,11 +107,21 @@ class NotificationHelper(private val context: Context) {
                 )
                 .build()
         }
+        // If we can't show all of our contacts as shortcuts, we move the important element to the
```

Comment:
Should we always move the important contacts to the start of the list?

Reply:
Done.